http-core/src/main/java/org/apache/pekko/http/javadsl/model/RemoteAddress.java

```diff
@@ -31,7 +30,7 @@ public abstract class RemoteAddress {
     public abstract int getPort();
 
     public static RemoteAddress create(InetAddress address) {
-        return org.apache.pekko.http.scaladsl.model.RemoteAddress.apply(address, OptionConverters.toScala(Optional.empty()));
+        return org.apache.pekko.http.scaladsl.model.RemoteAddress.apply(address, Util.convertOptionalToScala(Optional.empty()));
```

Comment:
this seems like a very roundabout way to get `scala.None` - with all the extra processing overhead of creating RichOptionals and converting them.

Reply:
Yes I am aware of this, I was trying to get the entire thing to compile. Setting up a PR in pekko core to add this method now.